x-pack/platform/test/api_integration_basic/config.basic_license.ts

```diff
@@ -5,28 +5,28 @@
  * 2.0.
  */
 
-/* eslint-disable import/no-default-export */
-
 import { FtrConfigProviderContext } from '@kbn/test';
 
 export default async function ({ readConfigFile }: FtrConfigProviderContext) {
-  const baseIntegrationTestsConfig = await readConfigFile(require.resolve('./config.ts'));
-  // security APIs should function the same under a basic or trial license
+  const xpackApiIntegrationConfig = await readConfigFile(
+    require.resolve('../api_integration/config.ts')
+  );
+
   return {
-    ...baseIntegrationTestsConfig.getAll(),
-    testFiles: [require.resolve('./apis/security/security_basic')],
+    // default to the xpack api integration config
+    ...xpackApiIntegrationConfig.getAll(),
     esTestCluster: {
-      ...baseIntegrationTestsConfig.get('esTestCluster'),
+      ...xpackApiIntegrationConfig.get('esTestCluster'),
       license: 'basic',
       serverArgs: [
-        ...baseIntegrationTestsConfig.get('esTestCluster.serverArgs'),
```

Comment:
>[!CAUTION]
> Are we sure we want to fully override `esTestCluster.serverArgs` from `xpackApiIntegrationConfig` instead of complementing it with these 3 additional values?

Reply:
good catch, we are losing extra arguments defined in `xpackApiIntegrationConfig`